addons/prometheusadapter/prometheusadapter.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: prometheusadapter
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "v0.7.0-3"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "v0.7.0-4"
     appversion.kubeaddons.mesosphere.io/prometheusadapter: "v0.7.0"
-    values.chart.helm.kubeaddons.mesosphere.io/prometheusadapter: "https://raw.githubusercontent.com/prometheus-community/helm-charts/a2b924bf7c439f0531634983dd941eeec2917725/charts/prometheus-adapter/values.yaml"
+    values.chart.helm.kubeaddons.mesosphere.io/prometheusadapter: "https://raw.githubusercontent.com/branden/prometheus-helm-charts/57c87ea696d270dca5df866c3195d851956ff27f/charts/prometheus-adapter/values.yaml"
```

Comment:
You should be able to use the standard repo: https://github.com/prometheus-community/helm-charts/commit/ef6391472c694c8cb1412b277f94ae3b7cfe972e

Reply:
done